### DISCUSSION TITLE
About“PK Analyses to CSV” can't export complete data.

### DISCUSSION THREAD

**Discussion by balamo98** [Open-Systems-Pharmacology/Forum#1457] [Questions & Problems]
Hello teams，
I created 100 individuals for the simulation, but when I exported PK data by“PK Analyses to CSV”, some data were missing. 
![图片](https://user-images.githubusercontent.com/96385989/220558548-2ef8936c-5137-4cfe-af70-5b4074007e7f.png)
Like this, there were only 93 or 87 individuals for my DDI simulations can be found.
Is there any way to deal with it？

Yours，colin

**Comment by PavelBal** [comment]
It seems that simulation of some individuals failed. This seems to be the same issue as here: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1449

@Yuri05 As this si a re-occuring topic, maybe this can be improved somehow (in the first place - find out why sampling of physiological values results in unsolvable simulations)

**Reply by Yuri05** [reply]
@balamo98 Can you attach your project?
